crates/dbsp/src/trace/ord/fallback/utils.rs

```diff
@@ -106,7 +99,7 @@ where
         B: BatchReader<Time = T>,
     {
         Self {
-            builder: O::Builder::new_builder(output_factories, T::default()),
+            builder: O::Builder::new_builder(output_factories),
```

Comment:
I wonder if we still need the merger API for anything. Definitely not for the merger itself.

Reply:
It would be pretty wonderful to drop thousands of lines of code of merger implementations!